### PR TITLE
feat: add new sql runner page with tables

### PR DESCRIPTION
**Pull request by joaoviana**
Relates to: #10551 

### Description:

- Adds new route `/sqlRunnerNew`
- Adds Sidebar (can be open and closed) - another PR will be open to handle the mount/unmount. We shouldn't do this and the `Transition` applied to the main Sidebar component should be replaced with a raw implementation 
- Gets `tables` from `useTables` hook and renders grouped by `schema`

NOTE: another PR will be open to handle the fields for each table

DEMO:

https://github.com/lightdash/lightdash/assets/7611706/88fc355f-6e5e-4e5b-93d2-3a89904a3580

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/Routes.tsx [hunk 1]
```diff
@@ -225,6 +226,12 @@ const Routes: FC = () => {
                                         </TrackPage>
                                     </Route>
 
+                                    <Route path="/projects/:projectUuid/sqlRunnerNew">
```
Comment (ZeRego) [inline packages/frontend/src/Routes.tsx:229]: Should we start using lowercase routes? I find it easier to read. 
```suggestion
                                    <Route path="/projects/:projectUuid/sql-runner-new">
```

File: packages/frontend/src/features/sqlRunner/components/Tables.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { LoadingOverlay, Stack, Text, UnstyledButton } from '@mantine/core';
+import { useState, type FC } from 'react';
+import { useTables } from '../hooks/useTables';
+
+type Props = {
+    projectUuid: string;
+};
+
+export const Tables: FC<Props> = ({ projectUuid }) => {
```
Comment (ZeRego) [inline packages/frontend/src/features/sqlRunner/components/Tables.tsx:9]: [not blocker] Would be nice to have a search for the tables.

**Review by ZeRego — APPROVED** [review]

File: packages/frontend/src/Routes.tsx (hunk 1 above)
Comment (joaoviana) [reply packages/frontend/src/Routes.tsx:229]: sure! happy to do that

File: packages/frontend/src/features/sqlRunner/components/Tables.tsx (hunk 2 above)
Comment (joaoviana) [reply packages/frontend/src/features/sqlRunner/components/Tables.tsx:9]: Nice! Will add search functionality to this and the fields on a separate PR

Comment (joaoviana) [reply packages/frontend/src/features/sqlRunner/components/Tables.tsx:9]: added here: https://github.com/lightdash/lightdash/issues/10569

**Pull request merged**